diff --git a/templates/form/form-element.html.twig b/templates/form/form-element.html.twig
index 9f40966..29edb1f 100644
--- a/templates/form/form-element.html.twig
+++ b/templates/form/form-element.html.twig
@@ -133,8 +133,13 @@
     {% if label_display in ['before', 'invisible'] %}
       {{ label }}
     {% endif %}
+    {% if (prefix is not empty) or (suffix is not empty) %}
+      <div class="input-group">
+    {% endif %}
     {% if prefix is not empty %}
-      <span class="field-prefix">{{ prefix }}</span>
+      <div class="input-group-prepend">
+        <span class="field-prefix input-group-text">{{ prefix }}</span>
+      </div>
     {% endif %}
     {% if description_display == 'before' and description.content %}
       <div{{ description.attributes }}>
@@ -143,7 +148,12 @@
     {% endif %}
     {{ children }}
     {% if suffix is not empty %}
-      <span class="field-suffix">{{ suffix }}</span>
+      <div class="input-group-append">
+        <span class="field-suffix input-group-text">{{ suffix }}</span>
+      </div>
+    {% endif %}
+    {% if (prefix is not empty) or (suffix is not empty) %}
+      </div>
     {% endif %}
     {% if label_display == 'after' %}
       {{ label }}
